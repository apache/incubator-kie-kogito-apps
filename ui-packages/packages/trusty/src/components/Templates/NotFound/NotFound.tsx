import React from 'react';
import {
  EmptyState,
  EmptyStateIcon,
  EmptyStatePrimary,
  PageSection,
  PageSectionVariants,
  TextContent,
  Title
} from '@patternfly/react-core';
import { WarningTriangleIcon } from '@patternfly/react-icons';
import { NavLink } from 'react-router-dom';

const NotFound = () => {
  return (
    <>
      <PageSection variant={PageSectionVariants.light}>
        <TextContent>
          <Title size="3xl" headingLevel="h2">
            Not Found
          </Title>
        </TextContent>
      </PageSection>
      <PageSection isFilled={true}>
        <EmptyState variant={'xl'}>
          <EmptyStateIcon icon={WarningTriangleIcon} />
          <Title size="2xl" headingLevel="h4">
            The page you are looking for does not exist
          </Title>
          <EmptyStatePrimary>
            <NavLink to="/">
              Go back to the &quot;Audit investigation&quot; home page
            </NavLink>
          </EmptyStatePrimary>
        </EmptyState>
      </PageSection>
    </>
  );
};

export default NotFound;
