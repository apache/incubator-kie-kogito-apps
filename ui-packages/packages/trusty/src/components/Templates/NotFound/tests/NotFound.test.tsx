import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../NotFound';

describe('NotFound', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
