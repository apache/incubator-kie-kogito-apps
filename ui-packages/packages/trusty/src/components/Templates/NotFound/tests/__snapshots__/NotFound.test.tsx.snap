// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<Fragment>
  <PageSection
    variant="light"
  >
    <TextContent>
      <Title
        headingLevel="h2"
        size="3xl"
      >
        Not Found
      </Title>
    </TextContent>
  </PageSection>
  <PageSection
    isFilled={true}
  >
    <EmptyState
      variant="xl"
    >
      <EmptyStateIcon
        icon={[Function]}
      />
      <Title
        headingLevel="h4"
        size="2xl"
      >
        The page you are looking for does not exist
      </Title>
      <EmptyStatePrimary>
        <NavLink
          to="/"
        >
          Go back to the "Audit investigation" home page
        </NavLink>
      </EmptyStatePrimary>
    </EmptyState>
  </PageSection>
</Fragment>
`;
